src/Sylius/Bundle/AdminApiBundle/Resources/config/app/config.yml

```diff
@@ -8,6 +8,7 @@ imports:
     - { resource: "@SyliusAdminApiBundle/Resources/config/grids/payments.yml" }
     - { resource: "@SyliusAdminApiBundle/Resources/config/grids/product.yml" }
     - { resource: "@SyliusAdminApiBundle/Resources/config/grids/product_variant.yml" }
+    - { resource: "@SyliusAdminApiBundle/Resources/config/grids/product_review.yml" }
```

Comment:
I guess that product_review should be before product_variant to keep alphabetical order